x/ibc/types.go

```diff
@@ -13,14 +13,14 @@ import (
 // IBCPacket defines a piece of data that can be send between two separate
 // blockchains.
 type IBCPacket struct {
-	SrcAddr   sdk.Address
-	DestAddr  sdk.Address
-	Coins     sdk.Coins
+	SrcAddr   bam.Address
+	DestAddr  bam.Address
+	Coins     bam.Coins
```

Comment:
What is the intention of moving address and coins etc. to baseapp instead of in types. Care to expand on your thinking a bit?